src/main/kotlin/app/revanced/patches/youtube/misc/hdrbrightness/fingerprints/HDRBrightnessFingerprint.kt

```diff
@@ -14,5 +18,18 @@ import app.revanced.patches.youtube.misc.hdrbrightness.annotations.HDRBrightness
 @Version("0.0.1")
 object HDRBrightnessFingerprint : MethodFingerprint(
     "V",
-    strings = listOf("c.SettingNotFound;", "screen_brightness", "android.mediaview", "lux"),
+    customFingerprint = { methodDef ->
```

Comment:
The custom fingerprint is very complex, instead a parent fingerprint should be used as previously mentioned.